options/posix/include/pthread.h

```diff
@@ -112,13 +112,15 @@ struct __mlibc_mutex {
 typedef struct __mlibc_mutex pthread_mutex_t;
 
 struct  __mlibc_condattr_struct {
-	// TODO: the clock attribute needs to be supported here.
+	int __mlibc_pshared;
+	clockid_t __mlibc_clock;
```

Comment:
Are we sure that this now marks the struct as complete?

Reply:
Yes, there's only set/get clock/pshared.